boards/common/gd32v/include/board_common.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2020 Koen Zandberg <koen@bergzand.net>
+ *               2023 Gunar Schorcht <gunar@schorcht.net>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_gd32v
+ * @{
+ *
+ * @file
+ * @brief       Common board definitions for GD32VF103 boards
+ *
+ * @author      Koen Zandberg <koen@bergzand.net>
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#ifndef BOARD_COMMON_H
+#define BOARD_COMMON_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "macros/units.h"
+
+/**
+ * @name    Xtimer configuration
+ * @{
+ */
+#define XTIMER_HZ                   MHZ(1)
```

Comment:
If I understand it right, you add it again only for the case if someone still uses xtimer instead of ztimer?

Reply:
Well ZTimer also takes its configuration from the XTimer config.
I could also directly use the new ZTimer config (`CONFIG_ZTIMER_USEC_WIDTH`) instead.

Reply:
`XTIMER_HZ =1MHz` is not needed (default) and most board do not set it if 1MHz
```suggestion
```